geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -11,42 +11,74 @@ Client application configuration
 ................................
 
 Previous versions had configuration in the `*.html.ejs`, in the `Controler*.js`, and in the `vars.yaml` file.
-The goal of this change is to move this configuration to the `vars.yaml` file (with default values in
+The goal of this change is to group all this configuration to the `vars.yaml` file (with default values in
 the `CONST_vars.yaml`). This will make future upgrades easier and increase configuration possibilities
 in the simple application mode, see the documentation
 <https://camptocamp.github.io/c2cgeoportal/${MAIN_BRANCH}/integrator/create_application.html#simple-application>.
 
 
-When you apply the `ngeo.diff` you should first apply the changes in the `geoportal/vars.yaml` file.
+When you apply the `ngeo.diff` you will have to remove lines of configuration from your `*.html.ejs` and
+`Controler*.js` files. You must compare each removed line of configuration with its default value newly
+defined in the section `vars/interface_config` of your `geoporal/CONST_vars.yaml` file. If you use a custom
+configuration, you have to override the default value by adding a new one in your `geoportal/vars.yaml`.
```

Comment:
For me, we can do a shortcut, if in the diff we see that we didn't change the default value we can apply this change without extra check...

Reply:
Proposale:

When you apply the `ngeo.diff` you will have to remove lines of configuration from your `*.html.ejs` and
`Controler*.js` files. If your configuration diverge from the default configuration you have to override the default value (newly defined in the `geoportal/CONST_vars.yaml`) by adding a new one in your `geoportal/vars.yaml`.

Reply:
+1 you just miss the precision: in the section `vars/interface_config` :-)

Reply:
Thanks I've adapted the text (and little bit more than what I wrote here, but the same idea)